src/connection/mod.rs

```diff
@@ -228,9 +241,15 @@ where
         self
     }
 
+    /// Charset. Default: UTF_8
+    pub fn charset(&mut self, charset: Charset) -> &mut Self {
```

Comment:
```suggestion
    /// Connection charset. It is only necessary to specify a charset other than the default `UTF-8` if you
    /// have text stored in the database using columns with charset `NONE` or `OCTETS`. Otherwise
    /// the database will handle the charset conversion automatically
    pub fn charset(&mut self, charset: Charset) -> &mut Self {
```
Just to tell that it is normally not needed to specify a charset.